autoPlay.user.js

```diff
@@ -251,7 +258,23 @@ function disableParticles() {
 	}
 }
 
+function updateCurrentTime() {
+    cHours = cTime.getUTCHours();
```

Comment:
This will always return the same value as `cTime` itself is never re-initialized.